lib/route.js

```diff
@@ -526,6 +528,7 @@ function preParsingHookRunner (functions, request, reply, cb) {
  * The purpose is to remove a socket from the tracked sockets collection when
  * the socket has naturally timed out.
  */
+/* istanbul ignore next */
```

Comment:
What is the issue here where the coverage needs to be ignored?

Reply:
Well, the thing is that is code might or might not be executed depending on which Node we run the test on the CI.